markdown/events/2022/bytesize-gitpod.md

```diff
@@ -25,3 +25,121 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[10:11](https://www.youtube.com/watch?v=kBoC6QBU-M0t=611)
+Now we're running nextflow and this pipeline within Gitpod. I hope you can see it's quite nice that we could quickly get to run this pipeline, see how it works and begin to see the output. The resources on these machines aren't huge, there's only a few threads, so you can't do anything that's going to be using, like, eight threads for example. They only run for maximum half an hour if you have the free account, but we'll explain later, that there's an access where you can get a limited time if you need that for your particular project.
+
+[10:46](https://www.youtube.com/watch?v=kBoC6QBU-M0t=646)
+If we just see that the program is running and we'll see all the tasks connected are running through, we can then... I'll just wait for it to finish, shouldn't take too long... This pipeline, I should have mentioned, is actually taking FASTQ reads. It's running it through FASTQC to get quality control of those particular samples and it's using MultiQC to push them all together to make a really nice plot of all the different features and the quality scores of the FASTQ reads.
+
+[11:20](https://www.youtube.com/watch?v=kBoC6QBU-M0t=680)
+This is a quick way of running a pipeline and you can do this for any of the nf-core pipelines that have this gitpod.yaml. It's quite fun to do. It means you can just really test out a new pipeline and see exactly how it works and if it works with your data. You can drop and drag your files into this explorer area to bring files in. There's also other ways that are normally here. There'll be a file in store, I don't know why it's not here. Might be because of the screen. Anyway, you can drop and drag files in here. I'll just take that off just see if we can find it. Yes, here we go. Sorry if I have the actual full screen. You have options here that you can save files to your local machine and you can open files and do all the kind of things you would expect to do in a normal computer.
+
+[12:06](https://www.youtube.com/watch?v=kBoC6QBU-M0t=726)
+The other thing, now it's finished. In our results file we have our results and if we look in MultiQC, for example, we have the MultiQC report. Again, you can use these preview buttons, so that you can render these particular results into a nice file where you can see everything that's going on. Here is the results of that FASTQC result in MultiQC. I think this is quite cool to know that you can you can just quickly run these pipelines.
```

Comment:
```suggestion
The other thing, now it's finished. In our results file, we have our results and if we look in MultiQC, for example, we have the MultiQC report. Again, you can use these preview buttons, so that you can render these particular results into a nice file where you can see everything that's going on. Here is the results of that FASTQC result in MultiQC. I think this is quite cool to know that you can just quickly run these pipelines.
```